Projects/Claims/Sources/CommonClaims/BulletPointCard.swift

```diff
@@ -28,10 +28,9 @@ extension BulletPointCard: Reusable {
         contentView.layoutMargins = UIEdgeInsets(top: 20, left: 10, bottom: 20, right: 20)
         contentView.isLayoutMarginsRelativeArrangement = true
         contentView.insetsLayoutMarginsFromSafeArea = false
-
         cardContainer.addSubview(contentView)
-
-        contentView.snp.makeConstraints { make in make.leading.equalToSuperview().inset(40)
+        contentView.snp.makeConstraints { make in
```

Comment:
should this file really be in claims or can we move it to hCoreUI?

Reply:
I think we are going to have a major change of UI over the summer and lots of things will be moved around and I feel it is not just right moment to take care of that.